packages/linter/src/executors/eslint/lint.impl.ts

```diff
@@ -39,6 +39,9 @@ export default async function run(
     ? join(options.cacheLocation, projectName)
     : undefined;
 
+  const printConfigForFile = options.printConfig;
```

Comment:
We can maybe use spread instead of mutating the original options

```suggestion
  const { printConfig, ...normalizedOptions } = options.printConfig;
```